e2e/test/prerequisites/readme.md

```diff
@@ -31,3 +31,26 @@ docker run -d --restart unless-stopped --name azure-iot-tinyproxy -p 127.0.0.1:8
 Alternatives:
  - TinyProxy for Linux/Mac: https://tinyproxy.github.io/
  - Squid for Windows: https://chocolatey.org/packages/squid
+
+ 
+### Far away hub
+In order to test reprovisioning, your dps instance under test must be linked to two different iot hubs. The first hub should already be set under the
+IOTHUB_CONN_STRING_CSHARP environment variable, and the other linked hub should be set as the value to the FAR_AWAY_IOTHUB_HOSTNAME environment variable
+
+While the name of this variable indicates that the hub is far away, the tests currently do not require the hub to be far away. The tests
+only require it to be a separate hub from the IOTHUB_CONN_STRING_CSHARP hub. It can be deployed to any region.
+
+
+### Custom allocation policy webhook
+
+Follow these instructions to setup a custom allocation webhook to test against:
+https://docs.microsoft.com/en-us/azure/iot-dps/how-to-use-custom-allocation-policies
+
+For the tests currently in this repo, your azure fucntion, which picks what iot hub to provision to, should always return the iot hub with the 
+longest hostname. Not all custom allocation policies need to work this way, but this is an easy-to-test functionality.
+
+In this folder, there is a run.csx file that is the code the azure function should run in order to achieve this behavior.
+
+The actual webhook url can be found through the azure portal, under your function, where there is a "Get Function URL" button that gives you a url 
+such as: "https://someazurefunction.azurewebsites.net/api/SomeTriggerName?code=XXXXXXXX" and this is the value that should be configured to the 
+CUSTOM_ALLOCATION_POLICY_WEBHOOK environment name
```

Comment:
Please update the cmd and sh templates as with these new variables.

Reply:
done